cohort-evaluator-spark/src/test/java/com/ibm/cohort/cql/terminology/R4FileSystemFhirTerminologyProviderTest.java

```diff
@@ -112,12 +199,24 @@ public void inOperationHandlesNullSystem() throws Exception {
 		boolean result = provider.in(code, info);
 		assertTrue(result);
 	}
+	
+	@Test(expected = IllegalArgumentException.class)
+	public void inOperationHandlesMultipleCodeSystemCode() throws Exception {
+		ValueSetInfo info = new ValueSetInfo();
+		info.setId(setId);
+
+		Code code = new Code();
+		code.setCode(TEST_CODE_MULTIPLE_CODE_SYSTEMS);
+
+		boolean result = provider.in(code, info);
+		assertTrue(result);
```

Comment:
Do you need this assertion still with the expected exception already a part of the test?

Reply:
removed, although I don't think it makes much difference